flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/BDREHDFSEventSink.java

```diff
@@ -122,6 +122,7 @@
   private SinkCounter sinkCounter;
 
   private volatile int idleTimeout;
+  @SuppressWarnings("squid:S1068")
   private Clock clock;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "clock" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1068)